src/Apps/Order2/Routes/Checkout/Components/Order2ReviewStep.tsx

```diff
@@ -41,7 +41,7 @@ const Order2ReviewStepComponent: React.FC<Order2ReviewStepProps> = ({
   const stripe = useStripe()
 
   // Get the offer ID for offer orders (only call the hook when needed)
-  const offerId = orderData.offers[0]?.internalID ?? null
+  const offerId = orderData.offers?.[0]?.internalID ?? null
```

Comment:
This shouldn't be allowed by the type system or any of the upstream systems - but I was getting a `orderData.offers is undefined` error. Restarting my dev server fixed it but i left the fix in.